spec/viewing_users_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+feature 'Viewing Users' do 
+    scenario 'I can see existing links on the links page' do
+    User.create(name: 'Jon', email: 'jon@example.com', password:'oranges')
+
+    visit '/users'
+
+    expect(page.status_code).to eq 200
+    expect(page).to have_content 'Jon'
+  end
```

Comment:
`end` at 11, 2 is not aligned with `scenario 'I can see existing links on the links page' do` at 4, 4